src/components/pages/career-details/career-form/career-file-upload-type.tsx

```diff
@@ -82,8 +82,13 @@ export const CareerFileUploadType = ({
   watch,
 }: CareerFileUploadTypeProps) => {
   const { t } = useTranslation();
-  const fileName = file.name.split('.');
-  const name = `category_select.${fileName[0]}`;
+  const fileName = file.name.split('.')[0];
```

Comment:
What does this do with file names that contain multiple dots like `submission.final.pdf`? It looks like it would discard the `final` in this example? Since this is only internal I guess it doesn't matter... but might lead to communication issues ("I've uploaded a file called abc.xzy.pdf, didn't you get it?"). I understand that it was like this before, but we might as well fix it now?

Reply:
Yeah the behavior was the same before. This is a hotfix, I just wanted to fix the fact that people can't upload files with comma in the name right now. It just fails and I have very limited time today. I would be happy if someone helps out to improve the PR in that direction or create a follow up PR 🙏